.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "4.4.2"
+  - 6
```

Comment:
Let's add Node.js 10 since it's the active LTS. 6 be removed since it isn't LTS; unless we need to support it?